### PR TITLE
Scale noise according to variable

### PR DESCRIPTION
**Pull request by mike-khor**
I noticed that with the new update to include V, pH, and Coxygen in GrowCHO.run output, the flex_sampling method adds noise to them "unequally", that is, because these 3 variables have very low absolute magnitudes, the stddev of 0.05 has a much larger impact on them as compared to actual flex2 variables. 2 things modified here:

- stddev of 0.05 is applied with a mean of 1, prior to being multiplied onto the variable. this way the noise is "scaled" depending on absolute magnitude of variable value. This way, low values won't appear more noisy compared to high values.
- Made volume variable `V` unaffected by noise, which is more in-line with how volume is tracked in most bioreactors.

Image below shows the "before" -- we have extremely noisy V, pH, and Coxygen
image

### CODE REVIEW

**Comment by svh-cb** [comment]
@amalhotra40 was also looking into this

what you did sounds fine to me and like a reasonable way to approach it, just want to make sure that it's the same that Abhinav had in mind before merge

File: insilicho/run.py [hunk 1]
```diff
@@ -207,12 +207,12 @@ def flex2_sampling(
...
         var = np.maximum(var, parameters.EPSILON)
-        if k == "time":
+        if k in ["time", "V"]:
             res[k] = var[idx].tolist()
         else:
             res[k] = np.maximum(
-                np.random.normal(
-                    loc=var[idx], scale=float(sampling_stddev), size=len(idx)
+                var[idx] * np.random.normal(
+                    loc=1.0, scale=float(sampling_stddev), size=len(idx)
```
Comment (amalhotra40) [inline insilicho/run.py:215]: 1. Just noting that L64 also tries to implement a similar outcome, i.e. to get values that are scaled (i.e. relative_std_dev) so maybe good to standardize the logic? 

2. I would rename args and docstring with prefix "rel" or "relative" ala L53 to make it clear the noise is relative.

**Review by amalhotra40 — APPROVED** [review]
Just two cosmetic changes to make things standardized, thanks for taking care of this.

File: insilicho/run.py (hunk 1 above)
Comment (mike-khor) [reply insilicho/run.py:215]: makes sense -- I could abstract the logic out to be applied to both parameters and flex2 measurements. good point about arg and docstring

File: tests/test_solver.py [hunk 2]
```diff
@@ -55,13 +10,13 @@ def test_solver_with_bolus_feed(self, bolus_feed: run.GrowCHO):
         assert bolus_feed.full_result.info["message"] == "Integration successful."
         # Ending concentrations
         assert (bolus_feed.full_result.state[-1, 0]) * 1e-9 == pytest.approx(
-            26.5, rel=0.005
+            26.1, rel=0.005
         )  # Xv
         assert (bolus_feed.full_result.state[-1, 1]) * 1e-9 == pytest.approx(
-            27.0, rel=0.005
+            26.7, rel=0.005
```
[the remark this answers is not in the document]
Comment (mike-khor) [reply tests/test_solver.py:16]: yep -- now that Y_amm_gln and Y_lac_glc are randomized, the numbers shift a little

File: insilicho/parameters.py [hunk 3]
```diff
@@ -61,8 +61,8 @@ class InputParameters(UnitValidationMixin):
     q_lac_max: typing.Union[float, str] = 0.2e-9  # mmol/cell/hour
 
     # Yield parameters
-    Y_amm_gln: float = 0.9  # -
-    Y_lac_glc: float = 0.25  # -
+    Y_amm_gln: typing.Union[float, str] = 0.9  # -
```
Comment (amalhotra40) [inline insilicho/parameters.py:64]: nice, maybe add units to the units_map too? (i.e. dimensionless if they get passed as a string?

Comment (mike-khor) [reply insilicho/parameters.py]: done!

**Review by amalhotra40 — APPROVED** [review]

**Pull request merged**